server/paymentProviders/thegivingblock/index.js

```diff
@@ -36,15 +36,13 @@ async function handleErrorsAndRetry(result, path, options = {}, account = null)
       logger.debug('Access token is invalid. Requesting a new one.');
       let error;
       try {
-        const { accessToken, refreshToken } = await login(GIVINGBLOCK_USERNAME, GIVINGBLOCK_PASSWORD);
+        const { accessToken, refreshToken } = await login(USERNAME, PASSWORD);
```

Comment:
You forgot to refactor here.

Reply:
No need to account.update as it's already done.

Reply:
But after login don't we have to update the `account` so that future requests can use that? Note that this login problem can occur for any request. Only in the refresh method we update the `account`. 🤔

Reply:
`login` should update the account like `refresh`, why do we have different strategies?

Reply:
@znarf : Ah, I get it now. I've corrected this to make it similar to `refresh`. Let me know if all looks good. 👍🏽